apps/api/src/app/events/e2e/digest-events.e2e.ts

```diff
@@ -927,8 +1046,133 @@ describe('Trigger event - Digest triggered events - /v1/events/trigger (POST)',
       _environmentId: session.environment._id,
       _templateId: template._id,
     });
+    const allJobsBackoff = await jobRepository.find({
+      _environmentId: session.environment._id,
+      _templateId: template._id,
+    });
+    expect(jobCount).to.equal(15);
+  });
+
+  it('should create multiple digests based on different nested digestKeys with backoff', async function () {
+    const postId = MessageRepository.createObjectId();
+    const postId2 = MessageRepository.createObjectId();
```

Comment:
Thank you very much for improving the testing suites with more solid and thorough tests. 👏🏻